ci/ci_smoke_test_c8y.sh

```diff
@@ -82,19 +74,19 @@ fi
 # Adding sbin seems to be necessary for non Raspberry P OS systems as Debian or Ubuntu
 PATH=$PATH:/usr/sbin
 
-echo "Disconnect old bridge"
-
-# Disconnect - may fail if not there
-sudo tedge disconnect c8y
-
-# From now on exit if a command exits with a non-zero status.
-# Commands above are allowed to fail
-set -e
+python3 -m venv ~/env-c8y-api
+source ~/env-c8y-api/bin/activate
+pip3 install c8y-api
+export C8YDEVICEID=$(python3 ./ci/find_device_id.py --tenant $C8YTENANT --user $C8YUSERNAME --device $C8YDEVICE --url $C8YURL)
 
-./ci/configure_bridge.sh
+# after calling the script, the ID should be there
+if [ -z $C8YDEVICEID ]; then
+    echo "Error: Please supply your Cumulocity device ID  name as environment variable C8YDEVICEID"
+    exit 1
+else
+    echo "Your device: HIDDEN"
```

Comment:
Shouldn't the script fail here? If the device id could not be retrieved, it'll eventually fail tests, right?

Reply:
Hi, it would fail by calling exit 1 in line 85. However, that only covered the cases when no ID was set.
I replaced the check for zero by a regex that expects a number. If we used a wrong device name the ID would have been the string returned by find_device.py.